### PR TITLE
Created Rule and Notification classes

### PR DESCRIPTION
**Pull request by EfJot314**

### CODE REVIEW

**Comment by EfJot314** [comment]
https://github.com/EfJot314/ZeusPol/tree/DataStructures

File: src/main/java/agh/edu/zeuspol/data_structures/Notification.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package agh.edu.zeuspol.data_structures;
+
+import javax.xml.crypto.Data;
```
Comment (socz3qqq) [inline src/main/java/agh/edu/zeuspol/data_structures/Notification.java]: I fear this xml import, is it necessary?

**Review by socz3qqq — APPROVED** [review]
For now it's sufficient

File: src/main/java/agh/edu/zeuspol/data_structures/Notification.java (hunk 1 above)
Comment (EfJot314) [reply src/main/java/agh/edu/zeuspol/data_structures/Notification.java]: The superfluous XML import, a source of your apprehension, has been eradicated, rendering it inconsequential. Your concerns are now moot as the import no longer poses any risk.

File: src/main/java/agh/edu/zeuspol/data_structures/ActionType.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package agh.edu.zeuspol.data_structures;
```
Comment (Pepesob) [inline src/main/java/agh/edu/zeuspol/data_structures/ActionType.java:1]: Names should be extended in my opinion

**Review by Pepesob — APPROVED** [review]

**Pull request merged**